cmd/agent/subcommands/run/command.go

```diff
@@ -449,7 +454,7 @@ func startAgent(
 
 	// Setup stats telemetry handler
 	if sender, err := demux.GetDefaultSender(); err == nil {
-		telemetry.RegisterStatsSender(sender)
+		pkgTelemetry.RegisterStatsSender(sender)
```

Comment:
We should add a `TODO` comment here to make sure this is eventually addressed in a component-style pattern.

Reply:
Will do 👍 
My team has plans to remove the stats sender entirely once the default agent telemetry work ships. 
Ill add a TODO that this is going away.